tools/clang_format_wrapper.py

```diff
@@ -93,7 +93,9 @@ def run_clang_format(file_list: List[str]) -> bool:
     out = check_output(command, text=True)
     logger.debug(out)
 
-    return False if out == "clang-format did not modify any files\n" else True
+    # If there's only a single line of output, we can assume that nothing was changed.
+    # This is not very clean, but better than hard-coding the expected message.
+    return out.index("\n") == out.rindex("\n")
```

Comment:
Are you sure that is always true?
I would prefer to lose this in favour of #2972
As a band aid and prepare a patch that calls clang-format with the changes lines only.

Reply:
Pretty sure - whenever a file is changed, there is some detailed output